Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -511,6 +511,7 @@ - (void)processTarget:(HYPFormTarget *)target
             NSArray *deletedIndexPaths = [self.formsManager hideTargets:@[target]];
             [self deleteItemsAtIndexPaths:deletedIndexPaths];
         } break;
+        case HYPFormTargetActionClear:
```

Comment:
I guess this needs implementation too, can't remember if we had tests for `processTarget`. =/

```objc
        case HYPFormTargetActionClear: {
            NSArray *updatedIndexPaths = [self.formsManager clearTargets:@[target]];
            [self reloadItemsAtIndexPaths:updatedIndexPaths];
        } break;
```